### PR TITLE
LPS-181903 LPS-188072 Scope Collection Provider

### PR DESCRIPTION
**Pull request by gustavolimav**
**Story**
https://liferay.atlassian.net/browse/LPS-181903

**Subtasks**
https://liferay.atlassian.net/browse/LPS-188072

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:search

File: modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/info/collection/provider/SXPBlueprintInfoCollectionProvider.java [hunk 1]
```diff
@@ -87,6 +100,29 @@ public InfoPage<AssetEntry> getCollectionInfoPage(
 			Collections.emptyList(), collectionQuery.getPagination(), 0);
 	}
 
+	@Override
+	public InfoForm getConfigurationInfoForm() {
+		return InfoForm.builder(
+		).infoFieldSetEntry(
```
[the remark this answers is not in the document]
Comment (gustavolimav) [reply modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/info/collection/provider/SXPBlueprintInfoCollectionProvider.java:109]: It cant be done in other order ex: https://github.com/liferay/liferay-portal/blob/064fa13921bfa4b21ba129a85897f18ec9f75ca2/modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java#L334-L345

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by felipelorenz** [comment]
LGTM!

**Comment by joshchong** [comment]
ci:forward:force

**Pull request closed without merging**